### ISSUE TITLE
PLEASE READ - Devs/Admin

### ISSUE THREAD

**Issue by paladinperry** [rg35xx-cfw/Koriki#122]
### What happened?

Good evening guys.

I'm a loyal user. If something is great. I have used Koriki ONLY since Koriki was launched.
It's the best looking and most user friendly OS for the RG35XX.
So thank you immensely.

I just think we would really like to know, if this is a dead project now ? or is there work going on still.

If it's dead. thank you so much for an epic OS.

If it's being worked on. thank you again. And please may we have an approx ETA for a new release ?

**Comment by acmeplus** [comment]
The project is not dead but has moved to https://knulli.org (and https://github.com/knulli-cfw). All custom firmware that I've been working during the last few years are now consolidated under that project.

There will be another update for the rg35xx but it will be initially EmulationStation based. The main reason is the initial goal is to have a consistent configuration across the different handhelds, and then focus on the differences.

**Comment by GithubUser99999999** [comment]
"The RG35XX is currently not supported by KNULLI."

So, is it dead now?

**Comment by terremoth** [comment]
> "The RG35XX is currently not supported by KNULLI."
> 
> So, is it dead now?

Yes, I think the first RG35XX was never supporteted by KNULLI and I think they never will. Guess this repo is the latest Koriki release for RG35XX unless @acmeplus or someone else create a new release.
